pkg/kuadranttools/authorino_tools.go

```diff
@@ -0,0 +1,71 @@
+package kuadranttools
+
+import (
+	"fmt"
+	"reflect"
+
+	authorinov1beta1 "github.com/kuadrant/authorino-operator/api/v1beta1"
+	"github.com/kuadrant/kuadrant-operator/api/v1beta1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func AuthorinoMutator(existingObj, desiredObj client.Object) (bool, error) {
+	update := false
+	existing, ok := existingObj.(*authorinov1beta1.Authorino)
+	if !ok {
+		return false, fmt.Errorf("existingObj %T is not a *authorinoauthorinov1beta1.Authorino", existingObj)
+	}
+	desired, ok := desiredObj.(*authorinov1beta1.Authorino)
+	if !ok {
+		return false, fmt.Errorf("desiredObj %T is not a *authorinoauthorinov1beta1.Authorino", desiredObj)
```

Comment:
nit typo in log:
```suggestion
		return false, fmt.Errorf("existingObj %T is not a *authorinov1beta1.Authorino", existingObj)
	}
	desired, ok := desiredObj.(*authorinov1beta1.Authorino)
	if !ok {
		return false, fmt.Errorf("desiredObj %T is not a *authorinov1beta1.Authorino", desiredObj)
```

Reply:
Going to blame copy and paste. Will fix.